scripts/apps/search/views/multi-image-edit.html

```diff
@@ -74,10 +74,15 @@ <h3 class="sd-slide-in-panel__heading" translate>Metadata</h3>
                         <div class="sd-alert sd-alert--hollow sd-alert--primary sd-alert--small" style="margin: -1rem 0 3rem;">
                             <span translate>Currently editing: {{getSelectedImages().length}} items</span>
                         </div>
+                        <div class="sd-alert sd-alert--hollow sd-alert--alert sd-alert--small" \
+                            style="margin: -1rem 0 3rem;"
+                            ng-if="getSelectedImages().length > 0 && !isSelectedAndLockedInCurrentSession()">
+                            <span translate>Editing disabled as item locked by another user.</span>
+                        </div>
                         <div sd-media-metadata-editor
                             data-item="metadata"
                             data-validator="validator"
-                            data-disabled="!getSelectedImages().length"
+                            data-disabled="!getSelectedImages().length || !isSelectedAndLockedInCurrentSession()"
```

Comment:
Can you extract the expression into a function and use it in both places? Here and in line 79.